pages/common/adb-reverse.md

```diff
@@ -0,0 +1,20 @@
+# adb reverse
+
+> Android Debug Bridge Reverse: reverse socket connections from an Android emulator instance or connected Android devices.
+> More information: <https://developer.android.com/studio/command-line/adb>.
+
+- List all reverse socket connections from emulator/device:
```

Comment:
Perhaps this would be more consistent with other existing pages?
```suggestion
- List all reverse socket connections from emulators and devices:
```